packages/medusa/src/interfaces/price-selection-strategy.ts

```diff
@@ -34,9 +40,11 @@ export abstract class AbstractPriceSelectionStrategy
   ): IPriceSelectionStrategy
 
   public abstract calculateVariantPrice(
-    variant_id: string,
+    variantId: string,
     context: PriceSelectionContext
   ): Promise<PriceSelectionResult>
+
+  public abstract onVariantsPricesUpdate(variantIds: string[]): Promise<void>
```

Comment:
todo: remove abstract, I don't think it should be required to implement that method. Otherwise it would force the users to copy past the original impl if they don't want to implement it. wdyt?

Reply:
You mean the entire method from the abstract class? In that case, the interface is not properly implemented.

Reply:
No just the ley word 😊

Reply:
ahh ok, added it in https://github.com/medusajs/medusa/pull/3553/commits/2bdbe9453763505f969b52ba6f2e3dc17ddeeae9